mlir/test/Target/LLVMIR/llvmir-intrinsics.mlir

```diff
@@ -112,6 +112,25 @@ llvm.func @cos_test(%arg0: f32, %arg1: vector<8xf32>) {
   llvm.return
 }
 
+// CHECK-LABEL: @hyperbolic_trig_test
+llvm.func @hyperbolic_trig_test(%arg0: f32, %arg1: vector<8xf32>) {
+  // CHECK: call float @llvm.sinh.f32
+  "llvm.intr.sinh"(%arg0) : (f32) -> f32
```

Comment:
```suggestion
  llvm.intr.sinh(%arg0) : (f32) -> f32
```
Is it necessary that this uses the generic format? The import tests do not seem to do so.

Reply:
Given we have no roundtrip tests I am in favor of using the pretty printed form here!